src/main/java/com/mentalhealthapi/mentalhealthapi/dao/DisorderServiceStub.java

```diff
@@ -1,6 +1,7 @@
-package com.mentalhealthapi.mentalhealthapi.service;
+package com.mentalhealthapi.mentalhealthapi.dao;
 
 import com.mentalhealthapi.mentalhealthapi.dto.Disorder;
+import com.mentalhealthapi.mentalhealthapi.service.IDisorderService;
```

Comment:
The above two files were refactored into the DAO folder. I believe the services folder should just be reserved for services and the Stubs should be included outside of that folder per our lessons in class. I believe this will help you with organization. 

https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
Service stubs are OK in the services package.